turbo.json

```diff
@@ -76,8 +76,10 @@
     "AVATARAPI_PASSWORD",
     "GIPHY_API_KEY",
     "GOOGLE_API_CREDENTIALS",
+    "GOOGLE_CALENDAR_API_KEY",
     "GOOGLE_LOGIN_ENABLED",
     "GOOGLE_WEBHOOK_TOKEN",
+    "HOLIDAY_CACHE_DAYS",
```

Comment:
not sure if we need to add this as an env var. what about just a const variable somewhere like constants.ts ?